### ISSUE TITLE
Fix Request: false positive in AvoidRecreatingDateTimeFormatter for Kotlin

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#617]
in fun: 
```kotlin 
val date = ZonedDateTime.now(ZoneOffset.UTC).format(DateTimeFormatter.RFC_1123_DATE_TIME)
```

**Comment by jborgers** [comment]
other examples:
```kotlin
testContext.parameters[responseKey] = now.format(DateTimeFormatter.ISO_DATE_TIME)

private val EXECUTION_DATE_FORMATTER = DateTimeFormatter.ofPattern(EXECUTION_DATE_FORMAT) // top level
```
